### DISCUSSION TITLE
How use useRuntimeConfig with nuxt/bridge

### DISCUSSION THREAD

**Discussion by chz** [nuxt/nuxt#18116] [Questions]
**Hello how use useRuntimeConfig inside docker? my nuxt bridge app doesn't see my config. But they are exist**

**Comment by wobsoriano** [comment]
Can you show your code?

**Reply by chz** [reply]
I used in nuxt.config.js 

```
publicRuntimeConfig: {
 CART_API_URL: process.env.CART_API_URL
}
```

when i build in docker and run nuxt cant see CART_API_URL

**Reply by wobsoriano** [reply]
how are you calling the config?

**Reply by chz** [reply]
in vuex this.$config.CART_API_URL

**Reply by chz** [reply]
in development its ok but inside docker not working but inside docker: 

typing: node > process.env.CART_API_URL i see value

**Comment by duyleekun** [comment]
You should have a default value at build time. Please try changing your nuxt.config into this

```
publicRuntimeConfig: {
 CART_API_URL: process.env.CART_API_URL || ''
}
```

**Reply by IgnisDa** [reply]
Post your Dockerfile
